core/src/test/java/com/yahoo/imapnio/async/data/MessageNumberSetTest.java

```diff
@@ -144,6 +144,78 @@ public void testBuildStringWith0LengthMessageNumberSets() {
         Assert.assertNull(MessageNumberSet.buildString(new MessageNumberSet[0]), "Result mismatched.");
     }
 
+    /**
+     * Tests buildMessageNumberSets(String) method.
+     */
+    @Test
+    public void testBuildMessageNumberSetsWithOneString() throws ImapAsyncClientException {
+        final String msgSetStr = "1";
+        final MessageNumberSet messageNumberSet = new MessageNumberSet(1, 1);
+        final MessageNumberSet[] messageNumberSets = MessageNumberSet.buildMessageNumberSets(msgSetStr);
+        Assert.assertNotNull(messageNumberSets, "Result mismatched.");
```

Comment:
Use appropriate message strings. Using "Result mismatched" for all assert statements causes confusion when a test fails.